ci/scripts/install_azurite.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+case "$(uname)" in
+  Darwin)
+    brew install node
+    npm install -g azurite
+    which azurite
+    ;;
+  MINGW*)
+    choco install nodejs.install
+    npm install -g azurite
+    ;;
+  Linux)
+    apt-get update
+    apt-get -y install npm
+    npm install -g azurite
+    which azurite
+    ;;
+esac
+echo "node version = $(node --version)"
+echo "azurite version = $(azurite --version)"
+AZURITE_DIR=${1}/azurite
+mkdir $AZURITE_DIR
+
+# Start azurite
+azurite --silent --location $AZURITE_DIR --debug $AZURITE_DIR/debug.log &
```

Comment:
Could you run `azurite` in test by `boost::process` not at here?
See also:

* https://github.com/apache/arrow/blob/master/cpp/src/arrow/filesystem/s3_test_util.cc#L24-L38
* https://github.com/apache/arrow/blob/master/cpp/src/arrow/filesystem/s3_test_util.cc#L91-L147
* https://github.com/apache/arrow/blob/master/cpp/src/arrow/filesystem/s3fs_test.cc#L182